src/modules/comments-api.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable operator-linebreak */
+import key from './api-key.js';
+
+const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${key}/comments/`;
+
+class CommentsApi {
+  get = (id) => {
+    const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${key}/comments?item_id=${id}`;
```

Comment:
:+1: Nice implementation to hide the key from users in GitHub. I suggest you include it on a .env file and access it using process. This is done by :point_right:  [dor.env](https://www.npmjs.com/package/dotenv-webpack)